pkgs/development/python-modules/enocean/default.nix

```diff
@@ -32,20 +37,21 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
+  pythonRemoveDeps = [ "enum-compat" ];
+
   dependencies = [
     beautifulsoup4
-    enum-compat
     pyserial
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
-
   pythonImportsCheck = [
     "enocean.communicators"
     "enocean.protocol.packet"
     "enocean.utils"
   ];
 
+  nativeCheckInputs = [ pytestCheckHook ];
```

Comment:
Why was this moved?

Reply:
Previous feedback on other PRs taught me to keep my arguments to `buildPythonPackage` in a certain order

Reply:
I also do it that way but I think it's more important to keep a diff clean.

Reply:
Sure, I'll revert it.

Reply:
I can also revert the changes in the parameter set, if you like.